### ISSUE TITLE
Flash Notice Above Create New

### ISSUE THREAD

**Issue by deep021985** [activescaffold/active_scaffold#597]
I have following AS config in controller for showing Create New form above list

`config.list.always_show_create = true`

Default message is customized in model
`def to_label	
    "Ticket for " + self.msetcl_user.emp_name.titleize + self.open_tickets_text
 end
`
After creating a new record, flash notice is displayed above list as well as above Create New form(refer screenshot).
How do I remove message above list and keep the one above Form.
Also I want to remove the default prefix 'Created' before message.

![flash_msg](https://user-images.githubusercontent.com/783535/40710977-ae3408fa-6417-11e8-990c-9985e2f8286f.PNG)

**Comment by scambra** [comment]
Get js response with web inspector and post here

**Comment by deep021985** [comment]
Which JS response? on web console I am getting  content.min.js  & on Network tab there are many JS files which I cant post here.

**Comment by scambra** [comment]
js response on network tab for form submit

**Comment by deep021985** [comment]
Sorry for delay.
I got this response on submit

![image](https://user-images.githubusercontent.com/783535/41276021-7967929e-6e40-11e8-82ff-0559f66ef4a8.png)

**Comment by scambra** [comment]
Sorry I guess there is no XHR response when form is submitted, because form is not loaded with XHR. When list renders create form, it renders messages partial inside form and above list. I guess messages partial should remove key from flash after rendering, so it isn't rendered twice.

Can you create active_scaffold_overrides/_messages.html.erb partial with this content and test it? Test with update and delete too.

```erb
<% for name in [:info, :warning, :error] %>
  <% if flash[name].present? %>
    <div class="<%= "#{name}-message message" %>">
      <%= display_message flash[name] %>
      <% if request.xhr? %>
        <a href="#" class="close" title="<%= as_(:close) %>"><%= as_(:close) %></a>
      <% end %>
    </div>
  <% end %>
<% end %>
```

**Comment by deep021985** [comment]
I am using the remote version

Gmfile.lock
GIT
  remote: git://github.com/activescaffold/active_scaffold.git
  revision: <some key>
  specs:
    active_scaffold (3.4.99)
      ice_nine (~> 0.11)
      rails (>= 4.0.5)

How can I override? Is there any way to do it through config?

**Comment by scambra** [comment]
No, no config for it, test it creating a _messages.html.erb partial in app/views/active_scaffold_overrides, then your partial will be used instead of the one included in gem.

```erb
<% for name in [:info, :warning, :error] %>
  <% if flash[name].present? %>
    <div class="<%= "#{name}-message message" %>">
      <%= display_message flash[name] %>
      <% flash.delete name %>
      <% if request.xhr? %>
        <a href="#" class="close" title="<%= as_(:close) %>"><%= as_(:close) %></a>
      <% end %>
    </div>
  <% end %>
<% end %>
```

When you confirm it's working, I will add to master branch, I don't know if it will default behaviour or optional with config.

**Comment by deep021985** [comment]
Great!! Its working. Now there is only one message on create new form
![image](https://user-images.githubusercontent.com/783535/41467180-1293c06c-70c3-11e8-9803-763f7767c481.png)
